Example Project/SampleApp/YMSampleHomeViewController.m

```diff
@@ -29,7 +28,16 @@ - (void)dealloc
 // This is called by clicking the login button in the sample interface.
 - (IBAction)login:(id)sender
 {
-    [[YMLoginController sharedInstance] startLogin];
+    /* Add your client ID here */
+    [[YMLoginClient sharedInstance] setAppClientID:@"APP CLIENT ID"];
```

Comment:
Maybe we should set these in the app delegate? We would just call startLogin here. I feel like that's the standard place I see this kind of thing.